Marlin/Configuration.h

```diff
@@ -815,16 +815,26 @@
  * A mechanical or opto endstop is used to check for the presence of filament.
  *
  * RAMPS-based boards use SERVO3_PIN.
- * For other boards you may need to define FIL_RUNOUT_PIN.
+ * For other boards you may need to define FIL_RUNOUT_PIN 0/1/2/3/4.
  * By default the firmware assumes HIGH = has filament, LOW = ran out
  */
-//#define FILAMENT_RUNOUT_SENSOR
```

Comment:
#  USE MY NAMES AND RENAME YOUR OWN MARLIN 2.0
'My names are serious and can be merged ' but i'm sure you will change something lollllllllllllll